src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -20,7 +23,7 @@
 /**
  * Extra data for Citizens.
  */
-public class CitizenData
+public class CitizenData implements ILocatable
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready